README.md

```diff
@@ -54,6 +55,23 @@ systemd checkout and the particleos checkout.
 To build a newer systemd, run `git pull` in the systemd repository followed by
  `mkosi -f sandbox meson compile -C build` and `mkosi -t none`.
 
+## Signing keys
+
+ParticleOS images are signed for Secure Boot with the user keys. To generate a new key,
```

Comment:
```suggestion
ParticleOS images are signed for Secure Boot with the user's keys. To generate a new key,
```
but maybe the "machine's keys" would make more sense.

Reply:
I think talking about _machine's keys_ can be confused with the factory keys